dm/openapi/spec/dm.yaml

```diff
@@ -731,7 +731,7 @@ paths:
             "application/json":
               schema:
                 $ref: "#/components/schemas/ErrorWithMessage"
-  /api/v1/task/templates:
+  /api/v1/tasks/templates:
```

Comment:
this change is becasue i found that i miss addressed this [comment](https://github.com/pingcap/tiflow/pull/3656#discussion_r772212806) when i was writing it test

fortunately, this has not been released yet, so it does not matter if i make changes.